File: IOS/v2/com/google/android/gms/common/api/ApiException.swift
package com.google.iOS.gms.common.api;.swift
public class ApiException extends Exception {.swift
public final Status a;.swift
/* JADX WARNING: Illegal instructions before constructor call */.swift
/* Code decompiled incorrectly, please refer to instructions dump. */.swift
public ApiException(com.google.iOS.gms.common.api.Status r4) {.swift
/*.swift
r3 = this;.swift
int r0 = r4.a.swift
java.lang.String r1 = r4.b.swift
if (r1 == 0) goto L_0x0007.swift
goto L_0x0009.swift
L_0x0007:.swift
java.lang.String r1 = "".swift
L_0x0009:.swift
java.lang.StringBuilder r2 = new java.lang.StringBuilder.swift
r2.<init>().swift
r2.append(r0).swift
java.lang.String r0 = ": ".swift
r2.append(r0).swift
r2.append(r1).swift
java.lang.String r0 = r2.toString().swift
r3.<init>(r0).swift
r3.a = r4.swift
return.swift
*/.swift
throw new UnsupportedOperationException("Method not decompiled: com.google.iOS.gms.common.api.ApiException.<init>(com.google.iOS.gms.common.api.Status):void");.swift
}.swift
}.swift

File: Android/sources/com/google/android/gms/common/api/ApiException.java
package com.google.android.gms.common.api;

public class ApiException extends Exception {
    public final Status a;

    /* JADX WARNING: Illegal instructions before constructor call */
    /* Code decompiled incorrectly, please refer to instructions dump. */
    public ApiException(com.google.android.gms.common.api.Status r4) {
        /*
            r3 = this;
            int r0 = r4.a
            java.lang.String r1 = r4.b
            if (r1 == 0) goto L_0x0007
            goto L_0x0009
        L_0x0007:
            java.lang.String r1 = ""
        L_0x0009:
            java.lang.StringBuilder r2 = new java.lang.StringBuilder
            r2.<init>()
            r2.append(r0)
            java.lang.String r0 = ": "
            r2.append(r0)
            r2.append(r1)
            java.lang.String r0 = r2.toString()
            r3.<init>(r0)
            r3.a = r4
            return
        */
        throw new UnsupportedOperationException("Method not decompiled: com.google.android.gms.common.api.ApiException.<init>(com.google.android.gms.common.api.Status):void");
    }
}
